app/views/docs/database_options.erbmd

```diff
@@ -113,17 +109,7 @@ RailsLambda:
 
 ### 4️⃣ The DATBASE_URL Environment Variable
 
-We now need to set the `DATBASE_URL` environment variable so Rails can connect to the RDS Proxy. The CDK stack in step 1️⃣ above created this string which contains everything from the DBs user/password to the host in an AWS Service called [Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html). Open up your `bin/build-rails` file and add the code below under the `RAILS_MASTER_KEY` line in the "Environments & Configuration" section. But please change `myapp-dburl` with the value from `MyDbProxyDbUrlParameterName` when you deployed your CDK stack above.
-
-```shell
-DATABASE_URL=$(aws ssm get-parameter \
-  --name "myapp-dburl" \
-  --with-decryption \
-  --query "Parameter.Value" \
-  --output text 2> /dev/null | echo "$(</dev/stdin)"
-)
-echo "DATABASE_URL=$DATABASE_URL" >> ".env.$RAILS_ENV"
-```
+We now need to set the `DATBASE_URL` environment variable so Rails can connect to the RDS Proxy. The CDK stack in step 1️⃣ above created this string which contains everything from the DBs user/password to the host in an Parameter Store as described in our <%= doc_link :environment_and_configuration %> guide.
```

Comment:
Typo: `DATABASE_URL`

Reply:
WOW! Life saver, I was about to reach out to Brian or something :)